packages/vscode-extension/src/debugging/DebugSession.ts

```diff
@@ -116,6 +117,7 @@ export class DebugSession implements Disposable {
   }
 
   public async startParentDebugSession() {
+    this.useParentDebugSession = true;
```

Comment:
I'm worried this might break if at some point we start the JS debugger before Parent, though it probably shouldn't break anything other than having two separate debug sessions visible in the debug panel.
Still, I think I'd prefer the approach from a previous commit, where this was an option passed to the ctor.
The way it is written now, calling `startParentDebugSession` changes the behaviour of all future JS sessions as well, even after `stop`ping this one, which I feel can be surprising to the caller.

Reply:
This should never be the case. Will add assert here to make sure we guard against it.